source/clock.rst

```diff
@@ -0,0 +1,143 @@
+Not only that, they can learn how to tell time in terms of day and date, which helps them understand what is meant by months and days.
+
+.. image :: ../images/Clock-img4.png
+
+Speaking Time
+:::::::::::::
+In an analog clock, the clock is divided into 12 sections and the hands move in a clockwise direction. The smaller hand gives the hour- for example, if the hour hand is pointing at 3, then it is currently 3 o'clock. In this case, the minute hand will be at 12, because it has made a complete cycle.
+If the minute hand is pointing to the clock number 1, you would say 5 minutes and this done by counting by 5s around the clock. However, if the minute hand is in between two clock numbers, you have to count by 5s until you get to the nearest clock number, and then count up by ones to figure out how many minutes you have.
+
+In the digital clock, the children can take divide the display into three sections, the hour, minute and seconds.
+
+The speaking time is given by the speaking mode, which tells time in 'o'clock', 'three forty-five PM' or midnight.
+
+Learning about Angles
+:::::::::::::::::::::
+Angles are an important concept that can be learnt through Clock, especially since the hour and minute hand can be dragged.
+The hour hand of an analog clock turns 360° in 12 hours or 0.5° per minute. The minute hand rotates through 360° in 60 minutes or 6° per minute.
+
+.. image :: ../images/Clock-img6.png
+
+Finding the Geographical North or South
+:::::::::::::::::::::::::::::::::::::::
+With the concept of telling time, children can find an analog watch and explore the concept of true North.
+
+In the Northern Hemisphere...
+
+1. Place the watch on a level surface, such as the ground, or hold it horizontal in your hand
+2. Point the hour hand at the sun
+3. Find the angle between the hour hand and the twelve o' clock mark on the watch to find the north-south line
+
+In the Southern Hemisphere...
+
+1. Place the watch on a level surface, such as the ground, or hold it horizontal in your hand
+2. Point the hour hand at the sun
+3. Find the angle between the hour hand and the twelve o' clock mark on the watch to find the north-south line
+
+The centre of the angle between the hour hand and twelve o'clock mark is the north-south line. If you don't know which way is north and which south, just remember that no matter where you are, the sun rises in the east and sets in the west. In the northern hemisphere the sun is due south at midday. If your watch is set to daylight saving time bisect the angle between the hour hand and the one o'clock mark instead.
+
+
+Modifying Clock
```

Comment:
I'm fine with developer documentation being in the git repository for the clock activity, but I don't think the learner or teacher audience needs this in detail.  Please remove the section, and make sure it is in the clock activity repository.  Leave behind a reference to the activity source code, where it can be found.